packages/test-runner-chrome/package.json

```diff
@@ -48,6 +48,7 @@
     "@web/test-runner-core": "^0.11.0",
     "@web/test-runner-coverage-v8": "^0.7.0",
     "chrome-launcher": "^0.15.0",
+    "puppeteer": "19.10",
```

Comment:
was this dep added intentionally? I think test-runner-chrome uses puppeteer-core for a specific reason (to avoid depending on puppeteer directly), can you check this with lars?

Reply:
iirc puppeteer-core depends on using the chrome version installed on the users machine, whereas puppeteer ships it with the package or something like that

Reply:
Ah! I think this is accidental then!